Bottom: d48abb1d1214e1255c982084181d0ecc75d57fa0
Top:    56d750a51d4b076095dce025b720ba5fbb647111
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-09 14:29:47 -0700

Refresh of python-fix

---
diff --git a/lldb/CMakeLists.txt b/lldb/CMakeLists.txt
index 7ddb552968c..db2a3121d48 100644
--- a/lldb/CMakeLists.txt
+++ b/lldb/CMakeLists.txt
@@ -38,6 +38,8 @@ endif()
 
 if (NOT LLDB_DISABLE_PYTHON)
   execute_process(
+    # Some python configurations will misbehave if you pass prefix='', so we pass
+    # 'foo' and then strip it off the returned prefix.
     COMMAND ${PYTHON_EXECUTABLE}
         -c "import distutils.sysconfig; print(distutils.sysconfig.get_python_lib(False, False, 'foo')[4:])"
     OUTPUT_VARIABLE LLDB_PYTHON_DEFAULT_RELATIVE_PATH
